src/components/FeatureViews/Scheduler/ScheduleForm.js

```diff
@@ -0,0 +1,130 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as calenderActions from '../../../actions/calenderActions';
+import PropTypes from 'prop-types';
+import React, { Component, useState } from 'react';
+import BackButton from "../../BackButton.js";
+import SessionActions from "./SessionActions";
+import '../../../theme/theme.scss';
+import './scheduler.scss'
+import { NavLink } from "react-router-dom";
+import FullCalendar from '@fullcalendar/react';
+import interactionPlugin from '@fullcalendar/interaction'
+import resourceTimelinePlugin from '@fullcalendar/resource-timeline';
+
+//Material UI Imports
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+
+import { Divider, LinearProgress, Typography, ListItem, RadioGroup } from "@material-ui/core";
+
+
+
+
+
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import Button from "@material-ui/core/Button";
+
+function ResourceForm(props) {
+
+    const { onClose, selectedValue, open } = props;
```

Comment:
Optional, but you can directly destructure when you declare the parameters:

    function ResourceForm({onClose, selectedValue, open}) {